test/integration/listener_test.rb

```diff
@@ -47,6 +47,21 @@ def test_missing_required_parameters
     assert_equal 'required_params_missing', node['code']
   end
 
+  def test_invalid_percent_encoding
+    #We pass a string and not a hash as a parameter due to
```

Comment:
minor nit: leave a space between the comment symbol and the start of the comment for readability

Reply:
Done :+1: